app/views/users/index.html.erb

```diff
@@ -1 +1,17 @@
-<h1>List of users</h1>
+<main>
+  <section>
+    <ul>
+      <% @users.each do |user| %>
+        <%= link_to user_user_posts_path(user) do %>
```

Comment:
- [ ] Clicking on a user should take a user to the "users show page"(/users/1) but I am instead taken to the "posts index page"(/users/1/user_posts).